opacclient/opacapp/src/main/java/de/geeksfactory/opacclient/frontend/AccountFragment.java

```diff
@@ -1374,6 +1226,22 @@ public void onCollapseAnimationEnd() {
         refreshage();
     }
 
+    private void setHtmlTextOrHide(String value, TextView tv) {
+        if (value != null) {
```

Comment:
In the best case where all APIs set a property to `null` if it is not available, this would be good enough. But that's probably not the case in all APIs. Should I change it?